src/System.Composition.AttributedModel/src/System.Composition.AttributedModel.csproj

```diff
@@ -6,7 +6,16 @@
     <RootNamespace>System.Composition</RootNamespace>
     <AssemblyName>System.Composition.AttributedModel</AssemblyName>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
+    <AssemblyVersion>1.0.31.0</AssemblyVersion>
```

Comment:
It's bumped up 1 patch version from the latest on nuget:

https://www.nuget.org/packages/Microsoft.Composition/